### ISSUE TITLE
Is HANA connection possible?

### ISSUE THREAD

**Issue by sergiocasero** [JetBrains/Exposed#1211]
Hi folks! First of all, thanks for this incredible lib, it's really helpful and I'm using it on my projects.

Recently, I've requested to develop a connector to a SAP HANA database. While investigating, I've noticed they have a JDBC driver: https://help.sap.com/viewer/0eec0d68141541d1b07893a39944924e/2.0.02/en-US/ff15928cf5594d78b841fbbe649f04b4.html

As far I know HANA is a relational database but it changes the way the data is saved. 

PD: I know that this is non a common use case, is just for asking you!

**Comment by Tapac** [comment]
@sergiocasero , never heard of it. Besides there is no available docker image (require auth on docker hub) to test it. 
If you have SAP HANA available somewhere you can try to copy-paste any of existing dialects make a few changes and run tests. 
Maybe it will show that HANA is compatible with Exposed and then you can provide us a PR.

**Comment by sergiocasero** [comment]
Good idea!! I'll try to test it this weekend, as far I know is pretty similar than mysql...
